GitUI/ScriptsEngine/ScriptOptionsParser.cs

```diff
@@ -370,6 +371,15 @@ private static string GetRemotePath(string url)
                     newString = SelectOneRef(currentTags);
                     break;
 
+                case "HEAD":
+                    newString = uiCommands.GitModule.GetSelectedBranch(setDefaultIfEmpty: true);
```

Comment:
❓ We do not want to get "(no branch)" here, do we?
```suggestion
                    newString = uiCommands.GitModule.GetSelectedBranch(setDefaultIfEmpty: false);
```

Reply:
true is "" so it shouldbe OK as is

Reply:
You did not miss anything other than commenting on the new param documentation that I relied on.

Reply:
> What have I missed?
> 
> ```
>         public static readonly string DetachedBranch = "(no branch)";
> 
>         public string GetSelectedBranch(bool setDefaultIfEmpty)
> 
>             return result.ExitedSuccessfully
>                 ? result.StandardOutput
>                 : setDefaultIfEmpty ? DetachedHeadParser.DetachedBranch : string.Empty;
> ```

You missed nothing. It's just that the param is IMHO terribly named and I definitely can't understand it correctly (and so the error also in the comments)

I understand "Default"  as "return (kind of) default value of a return string type"
whereas it should be understood as "return a default/placeholder value (which is not empty)"